### PR TITLE
Added Photo upload capability to Speaker and Sponsor Forms

### PR DESCRIPTION
**Pull request by aditya1702**
Added browse button and upload handling code for uploading images on Sponsor and Speaker forms

### CODE REVIEW

**Comment by rafalkowalski** [comment]
Awesome :D

File: open_event/helpers/data.py [hunk 1]
```diff
@@ -140,6 +140,9 @@ def create_speaker(form, event_id):
         :param form: view data form
         :param event_id: Speaker belongs to Event by event id
         """
+        file = request.files["photo"]
+	filename = secure_filename(file.filename)
```
Comment (leto) [inline open_event/helpers/data.py]: Please make these lines have the same indentation as the others

File: open_event/helpers/data.py [hunk 2]
```diff
@@ -190,6 +194,9 @@ def create_sponsor(form, event_id):
         :param form: view data form
         :param event_id: Sponsor belongs to Event by event id
         """
+        file = request.files["logo"]
+    	filename = secure_filename(file.filename)
```
Comment (leto) [inline open_event/helpers/data.py]: Fix indentation

File: open_event/helpers/data.py [hunk 3]
```diff
@@ -216,6 +223,7 @@ def remove_sponsor(sponsor_id):
         :param sponsor_id: Sponsor id to remove object
         """
         sponsor = Sponsor.query.get(sponsor_id)
+    	os.remove(os.path.join(os.path.realpath('.') + '/static/Sponsor Logo/', sponsor.logo))
```
Comment (leto) [inline open_event/helpers/data.py]: Indentation

File: open_event/helpers/data.py [hunk 4]
```diff
@@ -190,6 +194,9 @@ def create_sponsor(form, event_id):
         :param form: view data form
         :param event_id: Sponsor belongs to Event by event id
         """
+        file = request.files["logo"]
+    	filename = secure_filename(file.filename)
+	file.save(os.path.join(os.path.realpath('.') + '/static/Sponsor Logo/', filename))
```
Comment (leto) [inline open_event/helpers/data.py]: Let's not have files/directories with spaces in them. I suggest "sponsor_logos" but anything without a space works for me. Also, indentation

File: open_event/templates/admin/model/speaker/list.html [hunk 5]
```diff
@@ -39,7 +39,7 @@
                         </td>
                         <td>{{speaker.id}}</td>
                         <td>{{speaker.name}}</td>
-                        <td>{{speaker.photo}}</td>
+                        <td><img src="/pic/Speaker Photos/{{speaker.photo}}" height="84" width="84"></td>
```
Comment (leto) [inline open_event/templates/admin/model/speaker/list.html]: I suggest "speaker_photos" for the directory name to avoid spaces.

**Comment by leto** [comment]
Great work @aditya1702 ! Remember, you can push more commits to the aditya1702-patch-7 branch and the Pull Request will reflect them. You don't need to make a new PR. Let me know if you have any questions.

**Comment by leto** [comment]
Great job!

**Pull request merged**